Source/WebKit/Scripts/PreferencesTemplates/WebPreferencesFeatures.cpp.erb

```diff
@@ -37,17 +38,25 @@ namespace WebKit {
 
 const Vector<RefPtr<API::Object>>& WebPreferences::features()
 {
-    static NeverDestroyed<Vector<RefPtr<API::Object>>> features(std::initializer_list<RefPtr<API::Object>> {
+    static LazyNeverDestroyed<Vector<RefPtr<API::Object>>> features;
```

Comment:
I get calling `InitializeWebKit2();` but why the switch to LazyNeverDestroyed + std::once_flag? This is main thread only code as far as I know.

Reply:
I think ideally this code would look like:
```
InitializeWebKit2();
static NeverDestroyed<Vector<RefPtr<API::Object>>> features(std::initializer_list<RefPtr<API::Object>> { ...
```

InitializeWebKit2() already does a std::call_once internally so I don't think we need another one here at the call site. Same comment applies to the other file changed in this PR.

Reply:
I figured it'd be good to be extra careful, but it sounds like this is unnecessary. I've reverted back to using `static NeverDestroyed` in the most recent push.